Owin.Security.Providers/Asana/AsanaAuthenticationOptions.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using Microsoft.Owin;
+using Microsoft.Owin.Security;
+
+namespace Owin.Security.Providers.Asana
+{
+    public class AsanaAuthenticationOptions : AuthenticationOptions
+    {
+        public class AsanaAuthenticationEndpoints
+        {
+            /// <summary>
+            /// Endpoint which is used to redirect users to request Asana access
+            /// </summary>
+            /// <remarks>
+            /// Defaults to https://app.asana.com/-/oauth_authorize
+            /// </remarks>
+            public string AuthorizationEndpoint { get; set; }
+
+            /// <summary>
+            /// Endpoint which is used to exchange code for access token
+            /// </summary>
+            /// <remarks>
+            /// Defaults to https://app.asana.com/-/oauth_token
+            /// </remarks>
+            public string TokenEndpoint { get; set; }
+
+            /// <summary>
+            /// Endpoint which is used to obtain user information after authentication
+            /// </summary>
+            /// <remarks>
+            /// Defaults to https://asana.com/1/OAuthGetRequestToken
+            /// </remarks>
+            public string UserInfoEndpoint { get; set; }
```

Comment:
You are welcome to make the change and submit a pull request. I probably won't do a release again just for that, but then at least it is fixed for the next time I am doing a release. 

Doubt it will make any difference to anyone who uses it however - as far as I know no-one messes around with those sort of settings.